tests/test_structural.py

```diff
@@ -355,6 +355,37 @@ def testClosenessCentrality(self):
         for obs, exp in zip(cl, expected_cl):
             self.assertAlmostEqual(obs, exp, places=4)
 
+    def testHarmonicCentrality(self):
+        g = Graph.Star(5)
+        cl = g.harmonic_centrality()
+        cl2 = [1.0] + [(1.0 + 1 / 2 * 3) / 4] * 4
+        for idx in range(g.vcount()):
+            self.assertAlmostEqual(cl[idx], cl2[idx], places=3)
+
+        g = Graph.Star(5)
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            cl = g.harmonic_centrality(cutoff=1.0)
+        cl2 = [1.0, 1.0, 1.0, 1.0, 1.0]
```

Comment:
I'm a bit confused here. Does this test asserts that with `cutoff=1.0`, all harmonic centralities for the star graph should be 1?

If so, that is incorrect. We should have `[1.0, 0.25, 0.25, 0.25, 0.25]`, as we take paths of length 1, but not not longer. Thus each of the "spoke nodes" can reach only the centre node, so the inverse distances of one spoke node are [1,0,0,0] which averages to 0.25. This is what I get with the Mathematica interface.

What am I misunderstanding? While I did fix some issues 9 days ago, those already seem to be picked up by python-igraph.

Reply:
<img width="447" alt="image" src="https://user-images.githubusercontent.com/1212871/107114801-0df5cf00-6869-11eb-89a7-b80816784e40.png">

Reply:
Indeed; this was also confirmed by the fact that the test case failed. I've updated the test case.